protocol/bc/output.go

```diff
@@ -1,26 +1,26 @@
 package bc
 
-type output struct {
+type Output struct {
```

Comment:
The exported identifiers really ought to have comments. Especially this one to explain how it's different from TxOutput.